src/main/java/io/tja/osrs/shattered_relics_fragment_presets/ShatteredRelicsFragmentPresetsConfig.java

```diff
@@ -36,13 +36,26 @@ default int resizableOffsetY() {
     }
 
     @ConfigItem(
-            keyName = "shift_click_Equip",
-            name = "Shift-click equip",
-            description = "Equip fragments by shift-clicking them. Note: only works when clicking fragment text, not " +
-                    "fragment icons.",
+            keyName = "shift_click_equip_unequip",
+            name = "Shift-click equip/unequip",
+            description = "Equip and unequip fragments by shift-clicking them",
             position = 2
     )
-    default boolean shitClickEquipFragment() {
+    default boolean shiftClickToggleFragment() {
         return false;
```

Comment:
I think I'm going to turn this on by default. only reason we didn't do that before is that it wasn't working very well